front/app/components/admin/NavigationTabs/Tab.tsx

```diff
@@ -78,8 +76,7 @@ type TabProps = {
   className?: string;
   'data-cy'?: string;
   label: string;
-  // url: string;
-  url: initiativeRouteTypes;
+  url: string;
   active: boolean;
```

Comment:
Just reset this for now. Once we get all the routes converted we can add back the new general "Route" type?